File: KeeprVande/Models/Keep.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KeeprVande.Models;

public class Keep : RepoItem<int>
{
  public string Name { get; set; }
  public string Description { get; set; }
  public string Img { get; set; }
  public int Views { get; set; }
  public int Kept { get; set; }
  public string CreatorId { get; set; }
  public Profile Creator { get; set; }
}

public class KeepVaultKeep : Keep
{
  public int VaultKeepId { get; set; }
}

File: KeeprVande.client/src/models/Keep.js
export class Keep {
  constructor(data) {
    this.id = data.id
    this.name = data.name
    this.img = data.img
    this.description = data.description
    this.kept = data.kept
    this.views = data.views
    this.creatorId = data.creatorId
    this.creator = data.creator
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
  }
}